gen/builders/state_tracker.go

```diff
@@ -23,37 +30,154 @@ import (
 // messages.
 type StateTracker struct {
 	vector *schema.TestVector
+	bc     *BuilderCommon
+
+	StateTreeVersion types.StateTreeVersion
+	ActorsVersion    actors.Version
```

Comment:
We store these versions in the StateTracker so that vector scripts can alter behaviour as needed.